js/join.js

```diff
@@ -314,11 +343,11 @@ submit.addEventListener('click', async () => {
     headers: {
       'Content-Type': 'application/json',
     },
-    body: data,
+    body: JSON.stringify(sendJoinData()),
   })
     .then((res) => res.json())
     .then((res) => {
-      if (res.statusCode !== 200) {
```

Comment:
Why are we getting rid of the status code here? This would name our error messaging inflexible and unmaintainable .

Reply:
I used statusCode instead of `status` made the changes